opts/options.go

```diff
@@ -5,13 +5,17 @@ import (
 
 	ds "github.com/ipfs/go-datastore"
 	dssync "github.com/ipfs/go-datastore/sync"
-	"github.com/libp2p/go-libp2p-protocol"
+	protocol "github.com/libp2p/go-libp2p-protocol"
 	record "github.com/libp2p/go-libp2p-record"
 )
 
-var ProtocolDHT protocol.ID = "/ipfs/kad/1.0.0"
-var ProtocolDHTOld protocol.ID = "/ipfs/dht"
-var DefaultProtocols = []protocol.ID{ProtocolDHT, ProtocolDHTOld}
+var (
+	ProtocolDHT protocol.ID = "/ipfs/kad/1.0.0"
+	// Deprecated: The old format did not support more than one message per stream, and is not
+	// supported or relevant with stream pooling. ProtocolDHT should be used instead.
+	ProtocolDHTOld   protocol.ID = "/ipfs/dht"
+	DefaultProtocols             = []protocol.ID{ProtocolDHT}
```

Comment:
We introduced the new protocol in year 2016: d09b2e73d0ad820b3fceb709bc25e545250eec4d

I think it's safe to decommission it without anything catching on fire, or without announcements or change management.